### PR TITLE
Add: Markdown and Markup Support for Support Tickets and Replies

### PR DESCRIPTION
**Pull request by martinmckenna**
## Description

Adds Markdown to all things support tickets

## Type of Change
- Breaking change ('break', 'deprecate')

## Applicable E2E Tests

To run relevant E2E tests, run these commands in 3 separate terminals:

1. `yarn && yarn start`
2. `yarn selenium`
3. `yarn e2e --spec=e2e/specs/support/smoke-tickets.spec.js`

### CODE REVIEW

File: src/components/Frame/Frame.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    //   </Frame>
+    // </NoSsr>
+  );
+};
+
+const styled = withStyles(styles, { withTheme: true });
+
+export default compose<CombinedProps, {}>(
+  styled,
+  React.memo
```
Comment (Jskobos) [inline src/components/Frame/Frame.tsx]: Should this be moved to the top? (although component isn't being used now)

File: src/components/core/Paper.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <FormHelperText className={classes.errorText}>{error}</FormHelperText>
+      )}
+    </React.Fragment>
+  );
+};
+
+const styled = withStyles(styles);
+
+export default compose<CombinedProps, PaperProps>(
+  // React.memo,
```
Comment (Jskobos) [inline src/components/core/Paper.tsx]: Should this be removed?

File: src/features/Support/SupportTicketDetail/TabbedReply/MarkdownReference.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+const MarkdownReference: React.FC<CombinedProps> = props => {
+  const { classes } = props;
+  return (
+    <div className={props.rootClass}>
+      <Typography variant="h2" className={classes.header}>
+        Tips
+      </Typography>
+      <Typography>
+        You can use Markdown to format your question. For more examples see this
```
Comment (Jskobos) [inline src/features/Support/SupportTicketDetail/TabbedReply/MarkdownReference.tsx]: This says 'question' but is also displayed on replies.
The tab name is "Reply" but is shown in the creation drawer (where the technical field name is `description` but that's pretty meaningless)

File: src/features/Support/SupportTicketDetail/TabbedReply/TabbedReply.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+});
+
+type CombinedProps = Props & ReplyProps & WithStyles<ClassNames>;
+
+const TabbedReply: React.FC<CombinedProps> = props => {
+  const { innerClass, rootClass, classes, value, error, ...rest } = props;
+
+  const tabs: Tab[] = [
+    {
+      title: 'Reply',
```
Comment (Jskobos) [inline src/features/Support/SupportTicketDetail/TabbedReply/TabbedReply.tsx]: Ought to be conditional based on the context.

File: src/components/Frame/Frame.tsx (hunk 1 above)
Comment (martinmckenna) [reply src/components/Frame/Frame.tsx]: nope. This styles was called `withTheme`, so order doesn't matter

**Comment by johnwcallahan** [comment]
Something I just noticed –

The Preview tab needs to have scrolling overflow:

Screen Shot 2019-05-16 at 9 47 29 AM

**Comment by johnwcallahan** [comment]
For ticket replies, in narrow viewports, we put the Markdown reference _under_ the Submit button. In the New Ticket drawer, we put in _above_ the Submit button. IMO it works better _under_ the button, so that the Submit button is easier to find.

**Pull request merged**